jupyter_events/logger.py

```diff
@@ -155,10 +155,19 @@ def _skip_message(record, **kwargs):
             It is always emitted with 'null', and we do not want it,
             since we are always emitting events only
```

Comment:
Could we update the doc string?
(For whatever reason, the suggestion functionality is not letting me select multiple lines outside the change area - so I'm manually adjusting the suggestion):
```suggestion
            """
            Conditionally remove 'message' from log record.
            By default, it is always emitted with 'null', and we do not want it,
            unless 'message' is defined in the schema.
            """
```

Reply:
:+1: done.